src/main/scala/org/constellation/consensus/Consensus.scala

```diff
@@ -410,14 +443,21 @@ class Consensus[F[_]: Concurrent](
     val proposalPercentage: Float = proposals.size * 100 / peerSize
     (proposalPercentage, proposals.size) match {
       case (0, _) =>
-        getOwnTransactionsToReturn.map(txs => Left(EmptyProposals(roundData.roundId, stage, txs)))
+        getOwnTransactionsToReturn.flatMap(
+          txs => getOwnObservationsToReturn.map(exs => Left(EmptyProposals(roundData.roundId, stage, txs, exs)))
+        )
       case (_, size) if size == 1 =>
-        getOwnTransactionsToReturn.map(txs => Left(EmptyProposals(roundData.roundId, stage, txs)))
+        getOwnTransactionsToReturn.flatMap(
+          txs => getOwnObservationsToReturn.map(exs => Left(EmptyProposals(roundData.roundId, stage, txs, exs)))
```

Comment:
I can see that you are checking if the size is equal to one and after this you return "EmptyProposals" 
is this the desired action?

Reply:
I haven't changed the existing functionalities and conditionals. Maybe it's right, cause if there is only my proposal it means the node has empty proposals from others. Need to check few lines above/below